examples/Example/src/bottomSheet/index.tsx

```diff
@@ -1,25 +1,47 @@
 import React, { Component } from 'react';
-import { Animated, StyleSheet, Text, View, Dimensions } from 'react-native';
+import {
+  Animated,
+  StyleSheet,
+  View,
+  Dimensions,
+  NativeSyntheticEvent,
+  NativeScrollEvent,
+} from 'react-native';
 import {
   PanGestureHandler,
   NativeViewGestureHandler,
   State,
   TapGestureHandler,
+  PanGestureHandlerStateChangeEvent,
+  PanGestureHandlerGestureEvent,
 } from 'react-native-gesture-handler';
 
 import { LoremIpsum } from '../common';
 import { USE_NATIVE_DRIVER } from '../config';
 
+type StateType = {
+  lastSnap: number;
+};
+
 const HEADER_HEIGHT = 50;
 const windowHeight = Dimensions.get('window').height;
 const SNAP_POINTS_FROM_TOP = [50, windowHeight * 0.4, windowHeight * 0.8];
-
-export class BottomSheet extends Component {
-  masterdrawer = React.createRef();
-  drawer = React.createRef();
-  drawerheader = React.createRef();
-  scroll = React.createRef();
-  constructor(props) {
+export class BottomSheet extends Component<{}, StateType> {
+  private masterdrawer = React.createRef<TapGestureHandler>();
+  private drawer = React.createRef<PanGestureHandler>();
+  private drawerheader = React.createRef<PanGestureHandler>();
+  private scroll = React.createRef<NativeViewGestureHandler>();
+  private _lastScrollYValue: number;
```

Comment:
do you need to prefix values with underscore in TS? I think this is not necessary at all

Reply:
Maybe when you have a similar variable elsewhere in code

Reply:
Every underscore in Examples perished after 4a246de.